docs/mine/lotus/seal-workers.md

```diff
@@ -18,6 +18,10 @@ The `lotus-worker` application should have been built and installed along with t
 
 ## Setting up the Miner
 
+### Choosing a data directory for the worker
+
+The Lotus Worker stores all the necessary data in the `~/lotusworker` folder. You can customize that by setting the `$LOTUS_WORKER_PATH` environment variable. It is recommended to put this folder on a NVMe SSD so that it gets faster disk performance.
```

Comment:
```suggestion
The Lotus Worker stores all the necessary data in the `~/.lotusworker` folder. You can customize that by setting the `$LOTUS_WORKER_PATH` environment variable. It is recommended to put this folder on a NVMe SSD so that it gets faster disk performance.
```

Reply:
I really screwed all the paths. I will grep for any wrong ones.